File: striver/S3L1/largest_element_in_a_array.java
package S3L1;

import java.util.Scanner;

public class largest_element_in_a_array {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int size = sc.nextInt();
        int[] arr = new int[size];
        for(int i=0;i<size;i++)
            arr[i] = sc.nextInt();
        System.out.println(le(arr));
    }
    public static int le(int[] arr){
        int max = arr[0];
        for(int i=1;i<arr.length;i++)
            if(max<arr[i])
                max = arr[i];
        return max;
    }
}

File: striver/S3L1/largest_element_in_a_array.py
size = int(input())
arr =  [0]*size
for i in range(0,size):
    arr[i]=int(input())
def le(arr):
    max = arr[0]
    for i in range (1,size):
        if arr[i] > max:
            max = arr[i]
    return max
print("largest element is ",le(arr))